### PR TITLE
Added apprentice alt-titles for Security and Engineering

### PR DESCRIPTION
**Pull request by Generalcamo**
From a suggestion by greenjoe.

Adds apprentice alt-titles for Security and Engineering. The Investigator Intern has a slightly different outfit removing the hazard vest and giving them normal shoes and forensic gloves like other Investigators. The Atmospherics Intern gets their special PDA and tools in the toolbelt. No other changes are given to them, and they have the same access as the standard cadet.

EDIT: This now also makes changes to medical alt-titles. Roles for Pharmacy Interns and Resident Psychiatrists have been added, and the Surgeon Intern has been renamed to the "Resident Surgeon" and given an age restriction similar to what investigator interns get. All medical alt-titles get similar outfit changes to their intended target roles.

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

File: html/changelogs/GeneralCamo - Cadet Alt Titles.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ChangeMe
```
Comment (Generalcamo) [inline html/changelogs/GeneralCamo - Cadet Alt Titles.yml:30]:
```suggestion
author: GeneralCamo
```

File: code/game/jobs/faction/idris.dm [hunk 2]
```diff
@@ -48,6 +48,7 @@
 		"Security Officer" = /datum/outfit/job/officer/idris,
 		"Warden" = /datum/outfit/job/warden/idris,
 		"Security Cadet" = /datum/outfit/job/intern_sec/idris,
+		"Investigator Intern" = /datum/outfit/job/intern_forensics/idris
```
Comment (Generalcamo) [inline code/game/jobs/faction/idris.dm:51]:
```suggestion
		"Investigator Intern" = /datum/outfit/job/intern_forensics/idris,
```

File: code/game/jobs/faction/pmc.dm [hunk 3]
```diff
@@ -41,6 +41,7 @@
 		"Security Officer" = /datum/outfit/job/officer/pmc,
 		"Warden" = /datum/outfit/job/warden/pmc,
 		"Security Cadet" = /datum/outfit/job/intern_sec/pmc,
+		"Investigator Intern" = /datum/outfit/job/intern_forensics/pmc
```
Comment (Generalcamo) [inline code/game/jobs/faction/pmc.dm:44]:
```suggestion
		"Investigator Intern" = /datum/outfit/job/intern_forensics/pmc,
```

**Review by alsoandanswer — APPROVED** [review]

File: code/game/jobs/job/security.dm [hunk 4]
```diff
@@ -249,6 +249,8 @@
 	departments = SIMPLEDEPT(DEPARTMENT_SECURITY)
 	department_flag = ENGSEC
 	faction = "Station"
+	alt_titles = list("Investigator Intern", "Warden Cadet")
```
Comment (NonQueueingMatt) [inline code/game/jobs/job/security.dm:252]: investigator intern needs an age minimum set to 24/25 years old

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/jobs/job/security.dm (hunk 4 above)
Comment (Generalcamo) [reply code/game/jobs/job/security.dm:252]: The medical intern does not have this, despite similar age restrictions existing for its jobs. Do you want this for that as well?

**Review by FluffyGhoster — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**